0.data_analysis/construct_phenotypic_expression_plate_4_fs_data/median_correlation_relationships/nbconverted/median_cross_corr.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Determine expression relationships between constructs
+# Correlations between aggregate replicates are compared
+
+# ## Imports
+
+# In[1]:
+
+
+import pandas as pd
+from pathlib import Path
+import sys
+
+
+# ## Find the git root Directory
+
+# In[2]:
+
+
+# Get the current working directory
+cwd = Path.cwd()
+
+if (cwd / ".git").is_dir():
+    root_dir = cwd
+
+else:
+    root_dir = None
+    for parent in cwd.parents:
+        if (parent / ".git").is_dir():
+            root_dir = parent
+            break
+
+# Check if a Git root directory was found
+if root_dir is None:
+    raise FileNotFoundError("No Git root directory found.")
+
+
+# ## Import helper correlation class
+
+# In[3]:
+
+
+sys.path.append(
+    f"{root_dir}/0.data_analysis/construct_phenotypic_expression_plate_4_fs_data/utils"
+)
+import create_correlations as cc
```

Comment:
Consider importing at beginning of the Notebook

Reply:
I thought about that, but I would have to define the block of code where I create the `root_dir` path above the imports to maintain that structure, which also relies on imports.